projects/js-packages/publicize-components/src/components/social-previews/modal.scss

```diff
@@ -4,6 +4,9 @@
 
 @import '@automattic/jetpack-base-styles/gutenberg-base-styles';
 
+$jp-gray: #dcdcde;
```

Comment:
If I use that variable without defining it here, the build throws error.

Reply:
Right, that's because `$jp-gray` doesn't exist in there. You'd have to pick a different gray:
https://github.com/WordPress/gutenberg/blob/a1f2679cfeec924c2d769b40575626820ac2cfbd/packages/base-styles/_colors.scss

Since we're in the block editor, I think it would make sense to use a gray that exists in the other elements of the interface.